terminologies/technique/fiberTractReconstruction.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/fiberTractReconstruction",

```

Comment:
maybe change this to "digitalVectorReconstraction" (vs. somewhere above "digitalVolumeReconstruction")

@tgbugs: what do you think?